src/Type/BooleanType.php

```diff
@@ -33,18 +34,37 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         if ($options['transform']) {
             $builder->addModelTransformer(new BooleanTypeToBooleanTransformer());
         }
+
+        if ('SonataFormBundle' !== $options['catalogue']) {
```

Comment:
Shouldn’t this be CoreBundle?

Reply:
What when we drop CoreBundle? We should copy translations or require core in 0.x. I will resolve this tomorrow.